paket.lock

```diff
@@ -4,269 +4,5 @@ GROUP Documentation
 RESTRICTION: || (== netcoreapp3.1) (== netstandard2.0) (== netstandard2.1)
 NUGET
   remote: https://api.nuget.org/v3/index.json
-    FSharp.Compiler.Service (34.1)
```

Comment:
this all got purged by paket when I did a paket update: I assume this was leftover from before the move to the fsharp.formatting tool